src/pages/views/PageDetails.tsx

```diff
@@ -59,22 +74,99 @@ export const PageDetails: React.FC<PageDetailsProps> = ({ id, params }) => {
     }
   };
 
+  const [
+    deleteAttributeValue,
+    deleteAttributeValueOpts
+  ] = useAttributeValueDeleteMutation({});
+
   return (
     <TypedPageRemove variables={{ id }} onCompleted={handlePageRemove}>
       {(pageRemove, pageRemoveOpts) => (
         <TypedPageUpdate>
           {(pageUpdate, pageUpdateOpts) => (
             <TypedPageDetailsQuery variables={{ id }}>
               {pageDetails => {
-                const handleUpdate = async (data: PageData) => {
+                const handleUpdate = async (data: PageSubmitData) => {
+                  let errors: Array<
+                    | AttributeErrorFragment
+                    | UploadErrorFragment
+                    | PageErrorFragment
+                  > = [];
+
+                  const uploadFilesResult = await Promise.all(
```

Comment:
Too much different stuff in one function. Would rather move that bit to a separate one.